gammasky/catalogs.py

```diff
@@ -10,50 +10,92 @@
     'make_tev_catalog_data',
     'make_3fgl_catalog_data',
     'make_2fhl_catalog_data',
-    'make_snrcat_catalog_data',
+    'make_snrcat_catalog_data'
 ]
 
 TO_JSON_KWARGS = dict(orient='split', double_precision=5)
 
 
 def make_tev_catalog_data(nrows=None):
-    # TODO: change to gamma-cat
-    click.secho('Skipping TeV catalog ... need to switch to gamma-cat', fg='red')
-    return
-
-    click.secho('Making TeV catalog data...', fg='green')
+    click.secho('Making TeV catalog data (from gamma-cat)...', fg='green')
 
     out_dir = Path('src/app/data/cat')
     out_dir.mkdir(parents=True, exist_ok=True)
 
-    url = 'https://github.com/gammapy/gamma-cat/blob/master/other_cats/tgevcat/tgevcat.ecsv?raw=true'
-    table = Table.read(url, format='ascii.ecsv')
-    cols = table.colnames
+    url = 'https://github.com/gammapy/gamma-cat/raw/master/docs/data/gammacat.fits.gz'
+    table = Table.read(url)
+    # gamma-cat already has a source_id column
+    cols = [
+        'source_id',
+        'common_name',
+        'gamma_names',
+        'other_names',
+        'ra',
+        'dec',
+        'glon',
+        'glat',
+    ]
+    # cols = table.colnames  <-- Not working right now because I get the error: Cannot convert a table with multi-dimensional columns to a pandas DataFrame.
```

Comment:
gamma-cat includes multi-dimensional columns for the SED data. As a temporary fix you could  exclude those e.g. using  `cols = [name for name in table.colnames if not 'sed' in name]`